js/pages/settings.js

```diff
@@ -534,6 +538,8 @@ const refreshUserDataAfterEdit = async () => {
   const updatedUserData = await getMyData();
   if (hasUserData(updatedUserData)) {
     userData = updatedUserData.data;
+    await firebase.auth().currentUser.reload();
+    firebaseAuthUser = firebase.auth().currentUser;
```

Comment:
in this case, `userData` is the firestore participant profile and `firebaseAuthUser` is the firebase auth profile. Both are globals, and updating them impacts the UI, so I think I need to leave this structure in place.
540 - assign the new participant profile data
541 - force reload the firebase auth data (it can't be directly assigned to the firebaseAuthUser)
542 - update the firebaseAuthUser